docs/Guide.md

```diff
@@ -61,7 +61,7 @@ Check out the [Components](./Components.md) documentation.
 
 ### Testing
 
-Ignite is preconfigured to have [Detox](https://github.com/wix/Detox) and Jest.
+Ignite is preconfigured to have [Maestro](https://maestro.mobile.dev/) and Jest.
```

Comment:
There is no real configuration since it's an external install, should we remove this note about maestro here?

Reply:
I think leaving a note about needing to install Maestro on your machine or CI would be warranted here

```suggestion
Ignite is preconfigured to use Jest for unit tests.

Ignite includes samples of UI end-to-end tests using [Maestro](https://maestro.mobile.dev/). See their docs on [Installing Maestro](https://maestro.mobile.dev/getting-started/installing-maestro) to run the flows.
```